about.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="navbar0">
+            <nav class="black-nav">
+                <ul class="black-nav-links">
+                    <li id="fb-li"><img id="fb" src="img/login/login_fb.png" alt="facebook icon"></li>
+                    <li id="tw-li"><img id="tw" src="img/login/login_tw.png" alt="twitter icon"></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </nav>
+        </div>
+        <div class="navbar1">
+            <div id="logo"><a href="index.html">ABDOU SHOP</a></div>
+            <div id="open-menu"><button id="open-menu-button" type="submit"><img src="img/header/open-menu.png" alt="open menu"></button></div>
+            <nav class="navbar">
+                <div id="close-menu"><button id="close-menu-button" type="submit"><img src="img/header/close-menu.png" alt="close menu"></button></div>
+                <ul class="navbar-links">
+                    <li><a class="navbar-link" id="first-link" href="index.html">HOME</a></li>
+                    <li><a class="navbar-link" id="second-link" href="#services">SERVICES</a></li>
+                    <li><a class="navbar-link" href="about.html">ABOUT</a></li>
+                    <li><a href="#"></a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <div id="about">
+        <h5>SHOP WHAT YOU LOVE FASTER AND EASIER.</h5>
+        <p class="h5-supportingText">Our store started on 08 May 2019 , in Algeria, Our goal is to let the customer enjoy differents types of clothes , in any categories , with good quality and cheap price than other stores</p>
+        <div class="questions">
+            <p>Please contact us per Email for any further questions about our store , or to be an affiliate marketer with us !</p>
+            <a href="mailto:belkaid.abdulah@gmail.com">belkaid.abdulah@gmail.com</a>
+        </div>
+    </div>
+    
+    <div class="maps">
+        <h1>Our Stores : </h1>
```

Comment:
- [ ] (OPTIONAL) Good job adding the about page. It would be great if the first heading on your page is an `h1` heading so as to follow the heading hierarchy. 
- [ ] (OPTIONAL) Also you are missing the `logo` section on your about page. It would be great if you could add that too 👍 
- Missing section:
![Annotation 2022-05-18 105946](https://user-images.githubusercontent.com/85113567/169014376-7a0270f1-3315-4fd7-a891-5aaae1fb924d.jpg)